packages/client-cli/cli.mjs

```diff
@@ -283,7 +283,7 @@ async function downloadAndProcess (options) {
   let config = options.config
   if (!config) {
     const configFilesAccessibility = await Promise.all(configFileNames.map(fileName => isFileAccessible(fileName)))
-    config = configFileNames.find((value, index) => configFilesAccessibility[index])
+    config = configFileNames.find((value, index) => configFilesAccessibility[index] && !value.startsWith('watt'))
```

Comment:
This is incorrect. You'll need to look at the schema itself unfortunately, because `platformatic.json` or `watt.json` are interchangeable.

Maybe it's better to put the updating of the config file behind an option, so it can be turned on/off manually.

Reply:
49d33d9